### ISSUE TITLE
Unable to run Selenide Tests on Selenoid

### ISSUE THREAD

**Issue by Soumendu-Khilar** [selenide/selenide#2493]
## The problem
I am trying to run a Selenide test remotely in Selenoid. I have change the Configuration.remote="http://localhost:4444/wd/hub". But when I run the test I see the below error message:

**org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Response code 500. Message: session not created: Chrome failed to start: exited normally.
  (session not created: DevToolsActivePort file doesn't exist)
  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)**  

## Details
I am running Selenoid using docker-compose.yml file. docker ps gives the selenoid container running status with the Container details.

## Tell us about your environment
* **Selenide Version**: 6.19.0
* **Chrome\Firefox\IE Version**: Chrome version 117.0
* **OS Version**: macOS

## Code To Reproduce Issue [ Good To Have ]
    @Test
    public void test1()
    {
        System.out.println("Hi");
        driver_init("chrome");
        open("https://www.google.com");
        WebDriverRunner.getWebDriver().manage().window().maximize();
    }

    public void driver_init(String browser) {
        switch (browser.toUpperCase()) {
            case "CHROME":
                //ChromeOptions
                ChromeOptions options = new ChromeOptions();
                options.addArguments("--remote-allow-origins=*");
                options.addArguments("enable-automation");
                options.addArguments("--no-sandbox");
                options.addArguments("--disable-infobars");
                options.addArguments("--disable-dev-shm-usage");
                options.addArguments("--disable-browser-side-navigation");
                options.addArguments("--disable-gpu");
                options.addArguments("--disable-extensions");
                options.addArguments("--ignore-ssl-errors=yes");
                options.addArguments("--ignore-certificate-errors");
                options.addArguments("disable-popup-blocking");

                Configuration.browserCapabilities = options;
                Configuration.remote="http://localhost:4444/wd/hub";
                break;
        }
    }

Do I need to provide the path for the chrome-driver binary manually?

**Comment by asolntsev** [comment]
> Do I need to provide the path for the chrome-driver binary manually?

@Soumendu-Khilar no, you don't need to provide the path for chromedriver. You don't even know this path because chromedriver runs in the Selenoid container.

**Comment by Soumendu-Khilar** [comment]
@asolntsev What could the causing this issue then and any possible resolution??

**Comment by asolntsev** [comment]
@Soumendu-Khilar I have no idea. I am not a Selenoid expert. 
But I am pretty sure this problem is not caused by Selenide. You should rather ask in some Selenoid chat/github etc.
